app/uk/gov/hmrc/trackingconsentfrontend/assets/src/castToArray.ts

```diff
@@ -0,0 +1,3 @@
+const castToArray = nodeList => Array.prototype.slice.apply(nodeList)
+
+export default castToArray
```

Comment:
I know @matthewmascord has a different solution to this.  We should standardise after one/both have hit master.